index.html

```diff
@@ -2,16 +2,45 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml">
   <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-    
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1'>
+    <meta name="apple-mobile-web-app-capable" content="yes">
+	<meta name="theme-color" content="#0082c9">
+    <link rel="shortcut icon" href="https://nextcloud.com/media/favicon-touch.png" />
+    <link rel="apple-touch-icon-precomposed" href="https://nextcloud.com/media/favicon-touch.png" />
     <title>Nextcloud Documentation</title>
+	<meta name="author" content="Nextcloud">
+	<meta itemprop="description" content="Nextcloud is an open source, self-hosted file share and communication platform. Access  &amp; sync your files, contacts, calendars  &amp; communicate and collaborate across your devices. You decide what happens with your data, where it is and who can access it!">
+	<meta name="description" content="Nextcloud is an open source, self-hosted file share and communication platform. Access  &amp; sync your files, contacts, calendars  &amp; communicate and collaborate across your devices. You decide what happens with your data, where it is and who can access it!">
+
+	<!-- Twitter Card data -->
+	<meta name="twitter:card" content="summary_large_image">
+	<meta name="twitter:site" content="@nextclouders">
+	<meta name="twitter:url" content="https://docs.nextcloud.com">
+	<meta name="twitter:title" content="Nextcloud Documentation">
+	<meta name="twitter:description" content="Nextcloud is an open source, self-hosted file share and communication platform. Access  &amp; sync your files, contacts, calendars  &amp; communicate and collaborate across your devices.">
+
+	<!-- Open Graph data -->
+	<meta property="og:title" content="Nextcloud Documentation" />
+	<meta property="og:type" content="article" />
+	<meta property="og:url" content="http://nextcloud.com" />
```

Comment:
For Twitter it is docs.nextcloud.com here it is only nextcloud.com?

Reply:
Good catch, not urgent - but - could you make a separate PR to fix it?